brainstormforce/astra
# AST-753 - Astra old-new builder migration postbox trigger cause JS error due to missing nonce

### Description
- Nonce was missing and loaded through `astra_notice_before_markup_astra-sites-on-active` but was missing from `admin_enqueue_scripts` which leads this issue

### Screenshots
- https://share.bsf.io/4gulYq6J

### Types of changes
- Bug fix 

### How has this been tested?
- Check Old->New header conversion postbox trigger working or not
- Check for free plugin install/activate trigger
- Check for any addon activate/deactivate case
- Check for bulk activate/deactivate case
- PS: Make sure to watch console every time
- Please also check notice actions working as we did before
- Task for the same - https://wp-astra.atlassian.net/browse/AST-738
- Check "Get Started" button events working correctly for install/activate Starter Templates plugin

### Checklist:
